src/shared.js

```diff
@@ -1562,14 +1562,16 @@ export const updateCollectionSettingData = async (biospecimenData, tubes, partic
 
     if (!settings[visit][conceptIds.bloodCollectionSetting]) {
         bloodTubes.forEach(tube => {
-            if(biospecimenData[tube.concept][conceptIds.collection.tube.isCollected] === conceptIds.yes) {
-
-                settings[visit][conceptIds.bloodCollectionSetting] = biospecimenData[conceptIds.collection.collectionSetting];
-
-                if(biospecimenData[conceptIds.collection.collectionSetting] === conceptIds.research) {
+            const tubeIsCollected = biospecimenData[tube.concept][conceptIds.collection.tube.isCollected] === conceptIds.yes;
+            const collectionSetting = biospecimenData[conceptIds.collection.collectionSetting];
```

Comment:
I mentioned this in the last PR and it was marked as "done"... We can declare some of these variables like `collectionSetting` and `isResearch` outside of these `for` loops